form-flow-valtimo/src/main/kotlin/com/ritense/valtimo/formflow/handler/FormFlowStepTypeFormHandler.kt

```diff
@@ -70,7 +70,10 @@ class FormFlowStepTypeFormHandler(
         val document = documentService.get(documentId)
         val documentContent = document.content().asJson() as ObjectNode
 
-        camundaFormAssociationService.prefillProcessVariables(formDefinition, document)
+        if (taskInstanceId == null) {
```

Comment:
I just did some testing: A user task form IO isn't being prefilled with 'regular' process variables

Reply:
What are "regular" process variables?

Reply:
never mind, it does seem to work